### PR TITLE
Cref example

### PR DESCRIPTION
**Pull request by nicholasjnelson**
Modified a version of Lydia's Jupyter notebook for creating a custom reference state so that is now loads a MESA profile and creates a custom reference state using the MESA model. In this case, the MESA model is a 1 M_sun zero-age main sequence star.

### CODE REVIEW

**Comment by gassmoeller** [comment]
@nicholasjnelson let's chat after lunch about how to bring this up-to-date with the current master.

File: examples/custom_reference_states/MESA-input-1Msun-ZAMS.ipynb [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+   "source": [
+    "file_write='cref_from_MESA.dat'\n",
+    "my_ref.write(file_write)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now you can use this file to run a Rayleigh model. Once your Rayleigh model has run you can use the ``reference`` and  ``transport`` files to check how your specified reference state looks when transfered into Rayleigh."
```
Comment (gassmoeller) [inline examples/custom_reference_states/MESA-input-1Msun-ZAMS.ipynb]: Can you provide an example input file in this folder that runs with the created files? That way users will have everything they need to run this in one place.

File: examples/custom_reference_states/mesa.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import numpy as np
+
+G = 6.67428e-8 # dyne cm^2 / g^2
```
Comment (gassmoeller) [inline examples/custom_reference_states/mesa.py]: Question to @feathern: What is our policy regarding SI vs other unit systems? Do you want to keep everything uniform? Is everybody allowed to use their systems?

File: examples/custom_reference_states/reference_tools.py [hunk 3]
```diff
@@ -0,0 +1,226 @@
+#Reference State Generation Tools  (reference_tools.py)
```
Comment (gassmoeller) [inline examples/custom_reference_states/reference_tools.py]: Please move this file over to post_processing with `git mv examples/custom_reference_states/reference_tools.py post_processing/reference_tools.py`.

File: examples/custom_reference_states/mesa.py (hunk 2 above)
Comment (feathern) [reply examples/custom_reference_states/mesa.py]: I think we allow people to use whichever units.  At least half of us use cgs, which is completely sensible :)  Besides, the code is unit-agnostic.

Comment (gassmoeller) [reply examples/custom_reference_states/mesa.py]: Not implying anything else :smile:. Ok then.

**Pull request merged**